yellowbrick/contrib/missing/base.py

```diff
@@ -51,6 +53,12 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
+        # Do not call super here - the data visualizer has been refactored
+        # to provide increased functionality that is not yet compatible with
+        # the current implementation. This mimicks the previous functionality.
+        # TODO: Refactor MissingDataVisualizer to make use of new features.
+        self.features_ = self.features
```

Comment:
Part of #923 was to ensure that no duplicate work occurred in subclasses; however, the missing data visualizers require specialized functionality. It is possible to update them to use the new functionality but would require a bit more research. Both this and scatter I simply updated to behave similarly to the old functionality.

Reply:
:+1: